homeassistant/components/deconz/config_flow.py

```diff
@@ -95,8 +94,7 @@ def __init__(self):
             self.bridges = []
 
         if len(self.bridges) == 1:
```

Comment:
We should abort if `len(self.bridges) == 0` ?

Reply:
No, I offer manual input as a backup in case no bridges are found